exercise_perseus_render/assets/src/vue/index.vue

```diff
@@ -1,23 +1,59 @@
 <template>
 
   <div id="exercise-container">
-    <perseus v-if="item" :item="item"></perseus>
+    <assessment-wrapper
+      v-ref:wrapper
+      v-if="exercise"
+      :itemId="itemId"
+      :masteryModel="exercise.mastery_model"
+      :masterySpacingTime="exercise.masterySpacingTime">
+      <perseus v-if="item" :item="item"></perseus>
+    </assessment-wrapper>
   </div>
 
 </template>
 
 
 <script>
 
-  const defaultQuestion = require('./question.json');
-
   module.exports = {
     data: () => ({
-      item: defaultQuestion,
+      item: undefined,
+      exercise: undefined,
+      itemId: undefined,
     }),
+    methods: {
+      setItemId() {
```

Comment:
who's supposed to call this setItemId( ) method? and also setItemData( ) method below?